xwalk.gyp

```diff
@@ -632,8 +634,11 @@
       ],
       'copies': [
```

Comment:
The pak files are being generated into `<(SHARED_INTERMEDIATE_DIR)/xwalk/locales` only because the value of `grit_out_dir` is set to that; what I'm asking is if there's any issue with setting it to `<(PRODUCT_DIR)/locales/xwalk` instead, so that you don't need to copy anything afterwards.